src/components/ResultCountLabel/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+
+import { ResultCountWrapper, ResultCountP } from "./styledComponents";
+import { ResultCountWrapperProps } from "./interface";
+
+const ResultCountLabel: FC<ResultCountWrapperProps> = ({ count }) => {
+  return (
+    <ResultCountWrapper
```

Comment:
Do you think those values has to be customisable from the outside? I wonder whether there is ever the case that flex direction or any other prop would be defined differently than right here? If never, then i assume that could be just written in styles, right? :)

Reply:
I mean all props, flexDirection, justifyContent, alignItems, padding? Is it fine to just permanently define them in the ResultCountWrapper styling, instead of passing them from the outside? :> We can also tak it through over the call.

Reply:
I've moved those values to styledComponents file :)